.travis.yml

```diff
@@ -1,2 +1,32 @@
+language: generic
+dist: xenial
 sudo: false
-script: exit 0
+before_install:
+- gem install bundler
+install:
+- bundle install
+before_script:
+- bundle exec hatchet ci:setup
+script: bundle exec rspec test/spec/
```

Comment:
Yup, no idea what happened there, and how `php` can even be missing... very unfortunate. The flappiness is annoying.

Parallelism would really have to be done e.g. once per PHP version, which right now gets its own `context` each... hmh.

Reply:
nb regarding parallelism...

`parallel_tests` just runs whole specs per process, but I'd need in particular the `context`s in this loop parallelized:
```ruby
context "with a composer.json" do
    ["5.5", "5.6", "7.0", "7.1", "7.2", "7.3"].select(&method(:php_on_stack?)).each do |series|
        context "requiring PHP #{series}" do
```

And `ci-queue` doesn't support the hooks that we need to re-use deploys:
> Because of how `ci-queue` execute the examples, `before(:all)` and `after(:all)` hooks are not supported. `rspec-queue` will explicitly reject them.
https://github.com/Shopify/ci-queue/tree/master/ruby#limitations

Reply:
tests now running in parallel; down to sub ten minutes